src/ocnicepost.fd/ocnicepost.F90

```diff
@@ -0,0 +1,577 @@
+program ocnicepost
```

Comment:
@DeniseWorthen 
At a minimum, please add a few lines of what this code does, what the inputs are, etc.
Note that the weight files have hard-wired names down in this code.  Please be sure to document that.
Also include the authorship, so users can know who to ask questions in the case an error is encountered.

Reply:
ok